galite-core/src/main/kotlin/org/kopi/galite/visual/ui/vaadin/block/SingleComponentBlockLayout.kt

```diff
@@ -24,7 +24,7 @@ import com.vaadin.flow.component.Component
 /**
  * A block layout that contains a single component inside.
  */
-class SingleComponentBlockLayout(val block: DGridBlock) : SimpleBlockLayout(1, 1) {
+class SingleComponentBlockLayout(override val block: DGridBlock) : SimpleBlockLayout(1, 1, block) {
```

Comment:
check in this class the method : fun setBlockAlignment
can you check if grid.addThemeVariants(GridVariant.LUMO_NO_BORDER) can remove border between rows?
also can we check the height of the align block